File: java/Driver.java
class Driver extends Account{
  String licenseDiver;
  

  public Driver(String licenseDiver, String name, String document){
    super(name, document);
    this.licenseDiver = licenseDiver;
  }

  void printDataDriver() {
    System.out.println("Driver "+ name + "; license: " + licenseDiver);
  }
}

File: js/Driver.js
class Driver extends Account {
  constructor(licenseCar, user, license){
    super(name, document)
    this.user = user
    this.license = license
    this.licenseCar = licenseCar
  }

  printDataDriver(){
    console.log(`Driver: ${this.user.name} | Car lisence: ${this.licenseCar}
    Driver License: ${this.license}`)
  }
}